mentii/LocalDB
# Books LocalDB

Add ability to populate local db with books table and data

**Significant Changes**
1. makefile now also creates a books table when run
2. Added .sh files to work with books table

**How to Test**
1. Make sure the books table is created when makefile is run
2. Check that putItem.sh can successfully populate a book into table
3. Make sure other scripts work too
